GraphQL-TSC.md

```diff
@@ -211,7 +211,7 @@ Each repository will be managed by the EasyCLA signature tool, ensuring that all
 
 GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). However when speaking on behalf of the GraphQL project (e.g. technical docs, project websites, READMEs, or anything merged into a git repository under the GraphQL organization) we do not allow posting opinions or messages of political advocacy using the voice of the project without prior approval via GraphQL TSC vote.
 
-Looking forward, the TSC will develop a simple set of guidelines on this topic for Foundation spaces. Our goal will be to ensure there is a clear and straightforward path for achieving consensus on the message, delivery, and follow-up on matters of social importance.
+In addition to prior approval from the TSC, any expressed opinion or message of political advocacy from the GraphQL project should meet these guidelines around message, delivery, and follow-through. All messages should be clear and coordinated across available surfaces and channels. Advocacy messages should be paired with tangible action from the project as well as a clear call-to-action for community members.
```

Comment:
This is a nit as I tripped up on the meaning on my first read through.

> should meet these guidelines around message, delivery, and follow-through.

I think this preamble is redundant.

Possibly easier to ensure messages achieve all of the requirements if they're presented as a "check list" of bullet points.
```
All political advocacy or opinion messages should be:
- clear
- coordinated across available surfaces and channels
- paired with tangible action from the project
- providing a clear call-to-action for community members
```

Reply:
This is great feedback. I'll incorporate. This is minor, but I'm hoping to avoid bullets so I'll try this as a comma list.